tests/data-files/tcl/clash_namespace.tcl

```diff
@@ -0,0 +1,60 @@
+namespace eval clash {
```

Comment:
Let's slap a licence on there so people can copy it and derive from it with peace of mind.

```suggestion
# Copyright   :  (C) 2021-2022, QBayLogic B.V.,
#                    2022     , Google Inc.
# License     :  BSD2 (see the file LICENSE)
# Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>

namespace eval clash {
```